src/auro-accordion.js

```diff
@@ -100,24 +133,33 @@ export class AuroAccordion extends LitElement {
     content.style.height = `${height}px`;
   }
 
+  updated(changedProperties) {
+    if (changedProperties.has('iconRight')) {
+      this.requestUpdate();
+    }
+  }
+
   // function that renders the HTML and CSS into  the scope of the component
   render() {
 
-    const chevronHtml = this.chevron
+    const chevronHtml = !this.noChevron
```

Comment:
```suggestion
    const chevronHtml = this.noChevron
      ? html`` : html`
      
        <div class="iconWrapper" part="chevron">
          ${this.generateIconHtml(this.expanded ? chevronUp.svg : chevronDown.svg)}
        </div>
      `;
```

If you just reverse the logic here you can get rid of the linter exclusion on line 16.